base/util_testing.go

```diff
@@ -197,15 +198,26 @@ func (b *TestBucket) DropDataStore(name sgbucket.DataStoreName) error {
 // 	return b.Bucket.DefaultDataStore()
 // }
 
+// rosmarUriFromPath works to convert a path to a rosmar uri.
+func rosmarUriFromPath(path string) string {
+	// convert windows paths to unix style paths for rosmar, plus leading /
+	uri := rosmar.URLScheme + "://"
+	if runtime.GOOS == "windows" {
+		if filepath.IsAbs(path) {
+			uri += "/"
+		}
+	}
+	return uri + strings.ReplaceAll(path, `\`, `/`)
```

Comment:
This should be inside the Windows-specific code. `\\` is a valid character in a filename on some other filesystems, and this would break such a path.

Reply:
I moved this code into rosmar, for clarity.